src/overlay/gather-markers.ts

```diff
@@ -21,6 +21,7 @@ import {
 import { NotebookElementFinder } from './element-finder';
 import { ICodeCellModel } from '@jupyterlab/cells';
 import { Widget, PanelLayout } from '@phosphor/widgets';
+import * as $ from 'JQuery';
```

Comment:
```suggestion
import * as $ from 'jquery';
```

For consistency with `log.ts` imports